testing/simulator/src/cli.rs

```diff
@@ -34,7 +34,7 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                         .short("s")
                         .long("speed_up_factor")
                         .takes_value(true)
-                        .default_value("4")
+                        .default_value("3")
```

Comment:
Oh yeah, this was an interesting one.. The initial seconds-per-slot was 6,000ms and when divided by 4 becomes `1500ms`.

When we use the `/config/spec` endpoint, our "millliseconds per slot" value gets turned into "seconds per slot" and back again. Due to this the VC gets a "milliseconds per slot" value of 1000ms instead of 1500ms and throws an error saying the BN is on a bad spec version.

It didn't quite seem worth it to fix this whole thing, so I took the easy route.

Reply:
Interesting! I'm happy with that resolution